src/test/java/core/strategy/operation/PurchaseHandlerImplTest.java

```diff
@@ -0,0 +1,60 @@
+package core.strategy.operation;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.models.FruitTransaction;
+import core.basesyntax.strategy.operation.PurchaseHandlerImpl;
+import org.junit.After;
+import org.junit.Test;
+
+public class PurchaseHandlerImplTest {
+
+    @Test
+    public void handle_enoughFruitsForPurchase_Ok() {
+        Storage.storage.put("kiwi", 243);
+        PurchaseHandlerImpl purchaseHandler = new PurchaseHandlerImpl();
```

Comment:
make it a field, createe it in methods annotated with Before annotation

Reply:
didn't understand what to do
The whole method with before?